.github/workflows/check.yaml

```diff
@@ -0,0 +1,11 @@
+name: check
+
+on:
+  pull_request:
+    branches: ['*']
```

Comment:
Consider removing this line as it's unnecessary.

Reply:
i prefer to be explicit